### PR TITLE
Adding the google one-tap authentication 

### PR DESCRIPTION
**Pull request by bluzeey**
The following PR adds in a mongodb connection to which the data about the user could be send through decoding the google-oauth-token generated in the frontend. A rough implementation, it requires the env variables to be protected and a helper function down the road for cross checking whether the user exists in the database.

### CODE REVIEW

**Comment by bluzeey** [comment]
Also google o-auth should added in the requirements.txt , if required down the line.

File: app.py [hunk 1]
```diff
@@ -11,6 +15,11 @@
 from sendgrid.helpers.mail import Mail
 from constants import ASSEMBO_CONTACT, sendgrid_templates
 
+cluster=MongoClient("mongodb+srv://SahilMaheshwari:sahil123@cluster1.ir5lh.mongodb.net/myFirstDatabase?retryWrites=true&w=majority",tls=True,
```
Comment (yumochi) [inline app.py]: we  need to move the url into env variable

File: app.py [hunk 2]
```diff
@@ -11,6 +15,11 @@
...
+                             tlsAllowInvalidCertificates=True)
+db=cluster["assembo"]
```
Comment (yumochi) [inline app.py:21]: I would move the name into env variable

**Review by yumochi — CHANGES_REQUESTED** [review]
Refer to my comments

File: app.py [hunk 3]
```diff
@@ -27,6 +36,18 @@ def after_request(response):
...
+@app.route("/login",methods=["GET"])
+def login():
+    token= request.args.get('Authorization')
+    decoded_token=jwt.decode(token, verify=False)
+    useremail=decoded_token['email']
+    username=decoded_token['given_name']
+    userpicture=decoded_token['picture']
+    userData={"name":username,"email":useremail,"profilePicture":userpicture}
+    collection.insert_one(userData)
+    print(decoded_token)
```
Comment (yumochi) [inline app.py]: please remove testing code here

**Review by yumochi — APPROVED** [review]

**Pull request merged**